### PR TITLE
Fix InventorySlot NPE

### PR DESCRIPTION
**Pull request by Wealthyturtle**
### Description

Adds a check for calling ``getItem()`` with the index ``-999``

---
**Target Minecraft Versions:**     Any
**Requirements:**     None
**Related Issues:**     #2306

### CODE REVIEW

File: src/main/java/ch/njol/skript/util/slot/InventorySlot.java [hunk 1]
```diff
@@ -62,6 +62,8 @@ public int getIndex() {
 	@Override
 	@Nullable
 	public ItemStack getItem() {
+		if(index == -999) //Non-existent slot, e.g. Outside GUI
```
Comment (TheLimeGlass) [inline src/main/java/ch/njol/skript/util/slot/InventorySlot.java]: `if (index == -999)` Skript always has spaces within it's if statements.

**Comment by Blueyescat** [comment]
clicking on outside of inventory doesn't mean you clicked on an air item

**Comment by TheLimeGlass** [comment]
> 
> 
> clicking on outside of inventory doesn't mean you clicked on an air item

True, it is actually null for reference, and in older versions like 1.8 I believe it returned null if the item clicked was air regardless.

**Comment by Wealthyturtle** [comment]
Would it be safe to return null in that case?

**Comment by Wealthyturtle** [comment]
Null seems fine to return, no NPEs thrown so I replaced returning air with returning null directly

**Pull request merged**